include/ahrs_driver.h

```diff
@@ -33,9 +33,9 @@ namespace FDILink
 #define PI 3.141592653589793
 #define DEG_TO_RAD 0.017453292519943295
 // Sample covariance
-#define IMU_MAG_COV 0.1
-#define IMU_GYRO_COV 0.1
-#define IMU_ACCEL_COV 0.5
+#define IMU_MAG_COV {0.01, 0.01, 0.01}
+#define IMU_GYRO_COV {0.01, 0.01, 0.01}
+#define IMU_ACCEL_COV {0.05, 0.05, 0.05}
```

Comment:
We probably don't need this. Just define the default values directly on the constructor.

Reply:
I kept this to conform with the previous code!

Reply:
I see. We will leave it that then.